test/sum.js

```diff
@@ -0,0 +1,4 @@
+function sum(a, b) {
+    return a + b;
+  }
+module.exports = sum;
```

Comment:
@LittleJennie 
@cminho921 

Two comments here:

1. Would recommend not committing and including any files that you have to test functionality (e.g., if Sum is a part of setting Jest up, but has nothing to do with your app, I would add them to your gitignore in the future).

2. This file and the next are missing newlines to be aligned (no pun intended) with the Airbnb style guide!